src/shared/selectors/flags.ts

```diff
@@ -45,6 +45,7 @@ export const OSS_FLAGS = {
   boardWithFlows: false,
   createWithFlows: false,
   leadWithFlows: false,
+  selfRemovalFromAccount: false,
```

Comment:
@hoorayimhelping It looks like we only use this flag in `UserListItem`, which as far as I can tell is not in OSS. Should we omit the attribute here?

Reply:
@lexun yep, if this isn't used in OSS, it can be removed.